force-app/main/default/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -232,7 +232,7 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
     * @param parentObjectField
     * @param detailRecords
     * @param parentKeyField
-    * @return Boolean true if lastParentId has no associated detail records in a GAU rollup
+    * @return Boolean false if lastParentId has no associated detail records in a GAU rollup
```

Comment:
Method documentation is more useful when it's correct, eh? 🙃

Reply:
documentation is like 🍕 even if it's not great, it's still pretty good